src/main/scala/de/heikoseeberger/sbtheader/SbtHeader.scala

```diff
@@ -29,7 +29,7 @@ object SbtHeader extends AutoPlugin {
   object autoImport {
 
     object HeaderPattern {
-      val cStyleBlockComment = """(?s)(/\*(?!\*).*?\*/(?:\n|\r|\r\n)+)(.*)""".r
```

Comment:
It's not a perfect definition, but a pragmatic approach that works well for ConductR and my pet projects. You can always use your own pattern, see the README for details. I won't include a pattern here which deletes JavaDoc or ScalaDoc.